README.md

```diff
@@ -134,6 +134,20 @@ You can Get and install the RISC-V toolchain from https://gnutoolchains.com/risc
 
 Go to litex-boards/litex_boards/targets and execute the target you want to build.
 
+For example, build a picorv32 on the ULX3SL:
+```
+cd ./litex-boards/litex_boards/targets
+./ulx3s.py --device LFE5U-85F --cpu-type picorv32
+```
+
+The soft CPU is called `top.bit` and should be in:
+
+`/litex-boards/litex_boards/targets/soc_basesoc_ulx3s/gateware`
+
+The BIOS firmware is called `bios.bin` and should be in:
+
+`/litex-boards/litex_boards/targets/soc_basesoc_ulx3s/software/bios`
```

Comment:
For more boards the bios.bin is also embedded inside the gateware. For a few very small devices like the iCE40 (or Spartan 6 LX9) the BIOS is stored on spiflash instead as there isn't enough ROM available inside the FPGA.